pfcpiface/up4.go

```diff
@@ -156,37 +159,25 @@ func (up4 *UP4) initCounter(counterID uint8, name string) error {
 	return nil
 }
 
-func resetCounterVal(p *UP4, counterID uint8, val uint64) {
+func releaseCounterID(up4 *UP4, p4counterID uint8, val uint64) {
```

Comment:
@Adityav369 can you use `up4 *UP4` as a function receiver, instead of as a function argument?

Reply:
Done.